source/extensions/filters/http/oauth2/filter.h

```diff
@@ -108,11 +108,14 @@ struct CookieNames {
               const std::string& oauth_expires)
       : bearer_token_(bearer_token.empty() ? "BearerToken" : bearer_token),
         oauth_hmac_(oauth_hmac.empty() ? "OauthHMAC" : oauth_hmac),
-        oauth_expires_(oauth_expires.empty() ? "OauthExpires" : oauth_expires) {}
+        oauth_expires_(oauth_expires.empty() ? "OauthExpires" : oauth_expires),
+        id_token_("IdToken"), refresh_token_("RefreshToken") {}
 
   const std::string bearer_token_;
   const std::string oauth_hmac_;
   const std::string oauth_expires_;
+  const std::string id_token_;
+  const std::string refresh_token_;
```

Comment:
Since id/refresh token are constants we end up paying for string copies unnecessarily every time we allocate this object.  How about something like `static constexpr absl::string_view IdToken = "IdToken"` on this class instead?